pyscriptjs/src/components/base.ts

```diff
@@ -166,7 +173,7 @@ export class BaseEvalElement extends HTMLElement {
             this.postEvaluate();
         } catch (err) {
             if (Element === undefined) {
-                Element = <Element>runtime.globals.get('Element');
+                Element = <Element>runtime.getGlobals().get('Element');
```

Comment:
Having the attribute feels more Pythonic. What's the reason you decided to go with an explicit getter/setter method?

Reply:
Hmm, all this is JS code, but yeah, attribute feels better, changing, thanks!